montecristo/src/main/kotlin/com/datastax/montecristo/model/versions/DatabaseVersion.kt

```diff
@@ -73,6 +76,7 @@ interface DatabaseVersion {
                     versionIdentifier.startsWith("6.0") -> DseV6X(versionIdentifier)
                     versionIdentifier.startsWith("6.7") -> DseV67X(versionIdentifier)
                     versionIdentifier.startsWith("6.8") -> DseV68X(versionIdentifier)
+                    versionIdentifier.startsWith("6.9") -> DseV69X(versionIdentifier)
```

Comment:
I really wish we had managed to figure out a way to automatically generate this based on a published resource.

Reply:
major versions…   this is the last major DSE version.  all new versions are now in HCD.  
when a new major HCD version comes out (>1.2) i will re-evaluate this.  maybe it's becomes obvious, maybe it all changes when it's on IBM download pages…

Reply:
I think thats the solution, its the download pages which will enable this.